admin/templates/files/file.html

```diff
@@ -0,0 +1,72 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load node_extras %}
+{% block title %}
+    <title>File</title>
+{% endblock title %}
+{% block content %}
+    <div class="container-fluid">
+        <div class="row">
+            <table class="table table-striped">
+                <h2>File: <b>{{ object.name }}</b></h2>
+                <thead>
+                    <tr>
+                        <th>Field</th>
+                        <th>Value</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>Guid</td>
+                        <td>{{ guid }}</td>
+                    </tr>
+                    <tr>
+                        <td>Name</td>
+                        <td>{{ object.name }}</td>
+                    </tr>
+                    <tr>
+                        <td>Provider</td>
+                        <td>{{ object.provider }}</td>
+                    </tr>
+                    <tr>
+                        <td>Version</td>
+                        <td>{{ version }}</td>
+                    </tr>
+                    {% if node_id %}
+                    <tr>
+                       <td>Parent</td>
+                        <td>
+                         <a href="{{ node | reverse_node }}">
+                            {{ node.title }}
+                        </a>
+                        </td>
+                    </tr>
```

Comment:
The file could belong to a project, a registration, or a preprint. Will this show properly for each? Also, what does the link go to? The admin page for the object would be ideal.